<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectiveLab</class>
 <widget class="QDialog" name="SelectiveLab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CIE L*a*b* Selection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="view">
       <property name="minimumSize">
        <size>
         <width>256</width>
         <height>256</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderView">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Selection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="6" column="0" colspan="3">
       <widget class="QCheckBox" name="checkBoxGuide">
        <property name="text">
         <string>Display guide</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QCheckBox" name="checkBoxStrict">
        <property name="text">
         <string>Strict selection</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="sliderCoverage">
        <property name="minimum">
         <number>-360</number>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="pageStep">
         <number>15</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="3">
       <widget class="QCheckBox" name="checkBoxPreview">
        <property name="text">
         <string>Preview effect</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelCoverage">
        <property name="text">
         <string>Coverage:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="spinBoxCoverage">
        <property name="suffix">
         <string>°</string>
        </property>
        <property name="minimum">
         <number>-360</number>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelHue">
        <property name="text">
         <string>Hue:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="spinBoxHue">
        <property name="suffix">
         <string>°</string>
        </property>
        <property name="minimum">
         <number>-181</number>
        </property>
        <property name="maximum">
         <number>181</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="sliderHue">
        <property name="minimum">
         <number>-181</number>
        </property>
        <property name="maximum">
         <number>181</number>
        </property>
        <property name="pageStep">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QCheckBox" name="checkBoxClipToGamut">
        <property name="text">
         <string>Clip to gamut</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SelectiveLab</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>710</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SelectiveLab</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>710</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxHue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderHue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>562</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>563</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxCoverage</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderCoverage</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>590</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>598</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderHue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxHue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>571</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderCoverage</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxCoverage</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>603</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>602</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderCoverage</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SelectiveLab</receiver>
   <slot>updateView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>598</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderHue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SelectiveLab</receiver>
   <slot>updateView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>569</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderView</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SelectiveLab</receiver>
   <slot>updateViewNoEmission()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>535</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxGuide</sender>
   <signal>toggled(bool)</signal>
   <receiver>SelectiveLab</receiver>
   <slot>updateViewNoEmission()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>634</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxPreview</sender>
   <signal>toggled(bool)</signal>
   <receiver>SelectiveLab</receiver>
   <slot>updateViewNoEmission()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>660</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxStrict</sender>
   <signal>toggled(bool)</signal>
   <receiver>SelectiveLab</receiver>
   <slot>updateView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxClipToGamut</sender>
   <signal>toggled(bool)</signal>
   <receiver>SelectiveLab</receiver>
   <slot>updateView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateView()</slot>
  <slot>updateViewNoEmission()</slot>
 </slots>
</ui>
